### ISSUE TITLE
AMQP proxy closes connection if message contains timestamp in ms

### ISSUE THREAD

**Issue by SORC3r3r** [cloudamqp/amqproxy#57]
We were consuming messages from RabbitMQ queues through the `amqpproxy`.

We noticed that we had one specific queue which caused our consuming application to loose the connection to RabbitMQ.

After some analysing we found that the messages in that particular queue contained a timestamp in ms as shown in the screenshot. 
We played around with the timestamp and the connection did not die if the timestamp was in seconds instead of milliseconds.

![Screenshot 2021-09-30 at 14 27 03](https://user-images.githubusercontent.com/22958825/135455299-cb68ff84-857a-4b7f-95d5-e4e0c45c936c.png)

Guess it would be nice to not close the connection if the timestamp is in milliseconds.

This is the error message the `amqpproxy` gave us:

```
amqp-proxy    | Unhandled exception in spawn: Invalid time: seconds out of range (ArgumentError)
amqp-proxy    |   from ???
amqp-proxy    |   from ???
amqp-proxy    |   from ???
amqp-proxy    |   from ???
amqp-proxy    |   from ???
```

**Comment by tezhm** [comment]
Experiencing same issue

**Comment by tezhm** [comment]
Never mind, issue was we were sending the timestamp in milliseconds when it is meant to be in seconds.

**Comment by carlhoerberg** [comment]
Fixed in 0.5.8, thank you for reporting.
